packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx

```diff
@@ -109,7 +109,7 @@ export const CreateTopicSection = () => {
           />
           <div className="description-char-count">
             <CWText type="caption">
-              {descriptionDelta.ops[0].insert.length} / 250
+              {descriptionDelta.ops?.[0].insert.length} / 250
```

Comment:
this can get to `NaN`, lets use this
```diff
- {descriptionDelta.ops?.[0].insert.length} / 250
+ ({descriptionDelta?.ops?.[0]?.insert?.length} / 250) || 0
```